homework/fibonacci/fibonacci.hpp

```diff
@@ -1,11 +1,28 @@
 #pragma once
 
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    int first = 0;
+    int second = 1;
+    if(!sequence){
+        return 0;
+    }
+    if(sequence == 1){
+        return 1;
+    }
+    for(int i=1;i<sequence;i++){
+        int result = first + second;
+        first = second;
+        second = result;
+    }
+    return second;
 }
 
 int fibonacci_recursive(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    if(!sequence){
+        return 0;
+    }
+    if(sequence == 1){
+        return 1;
+    }
```

Comment:
Sequence hold exactly the same value as you want to return for 0 and 1 :)

Better write:

`if (sequence < 2) {
    return sequence;
}`